Library/ViewModels/ProjectPageViewModel.swift

```diff
@@ -538,6 +541,23 @@ public final class ProjectPageViewModel: ProjectPageViewModelType, ProjectPageVi
         return (project, refTag, updatedValues)
       }
 
+    // TODO: SPC - Call new Similar Projects GraphAPI service method when ready
+    self.updateSimilarProjectsInDataSource = freshProjectAndRefTag
+      .map { freshProjectAndRefTag in
+        let (project, _) = freshProjectAndRefTag
+        let validProjectFragment = createMockProjectNode()
+
+        guard let similarProject = SimilarProjectFragment(validProjectFragment.fragments.projectCardFragment)
+        else { return (
+          project,
+          nil
+        ) }
+
+        // Return 'nil' for now until the feature flagging code is available.
+        return (project, nil)
+//        return (project, [similarProject, similarProject, similarProject, similarProject])
+      }
```

Comment:
temporary test projects being returned here until the new GraphAPI call is ready to replace.